.cargo/bin/fullstack.rs

```diff
@@ -0,0 +1,33 @@
+mod backend;
+mod frontend;
+mod net;
+mod tsync;
+
+pub fn main() {
```

Comment:
We decided that backend will host the frontend (without additional server), so it looks like this command is not needed

Reply:
3 files - backend.rs, frontend.rs and fullstack.rs - were needed exclusively for developing purposes. Come to think of this, there is no really purpose to push those into the repo. 

Resolve: Deletion